src/Test/Util.hs

```diff
@@ -1,22 +1,41 @@
-{-# LANGUAGE RecordWildCards, GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE RecordWildCards, GeneralizedNewtypeDeriving, StrictData #-}
```

Comment:
Can we just put ! on individual fields. I've been burnt by StrictData in the past.

Reply:
Done!